index.js

```diff
@@ -0,0 +1,2 @@
+export {Server} from './lib/server.js'
+export * from './lib/constants.js'
```

Comment:
```suggestion
export {DEFAULT_HEADERS, REQUEST_HEADERS, RESPONSE_HEADERS, IMAGE_MIME_TYPES} from './lib/constants.js'
```

Beter to use actual export identifiers than a `*`

Should all these constants be exported? For what?